jenkins-scripts/lib/windows_library.bat

```diff
@@ -194,7 +197,7 @@ set COLCON_PACKAGE=%1
 :: two runs to get the dependencies built with testing and the package under
 :: test build with tests
 echo # BEGIN SECTION: colcon compilation without test for dependencies of %COLCON_PACKAGE%
-call :_colcon_build_cmd --packages-skip %COLCON_PACKAGE% " -DBUILD_TESTING=0"
+call :_colcon_build_cmd --packages-skip %COLCON_PACKAGE% "-DBUILD_TESTING=0" "-DCMAKE_CXX_FLAGS=-w"

```

Comment:
out of curiosity, does it work if you put multiple arguments in a single string? (`"-DBUILD_TESTING=0 -DCMAKE_CXX_FLAGS=-w"`

Reply:
Nope :confused: See the note on line 163